### DISCUSSION TITLE
Field condition

### DISCUSSION THREAD

**Discussion by imbwaldo** [spatie/statamic-responsive-images#227] [General]
I'm trying to set a condition on a field (_Banner Headline_ in the screenshot) in the Statamic CP, such that the field will be hidden if no image is assigned to a particular responsive image field (_banner_image_responsive_ in the screenshot below). I can't find an option that works. Is it even possible to do this at the moment? If so, how?

<img width="677" alt="Screenshot 2023-05-15 at 5 02 40 PM" src="https://github.com/spatie/statamic-responsive-images/assets/1582379/8fce5717-509c-4fb6-a522-e96f242513a6">

**Comment by ncla** [comment]
I am writing this in theory, I do not have access to Statamic rn, but try targetting `src` property, so something like `banner_image_responsive.src`

**Reply by imbwaldo** [reply]
Thanks, @ncla. I tried that and, unless I did something incorrectly, it didn't work. But I found another—and better—way to achieve what I was after, so I'll file that suggestion away and try again if an opportunity presents itself.

**Reply by ncla** [reply]
What was the better, other way?

If responsive field does not work, you can utilize asset fieldtype as well. You can pass asset field value to responsive tag in Antlers, and it will work just like before. Unless you are utilizing other fields like ratio, fit, multiple breakpoints then it becomes bigger hassle. But a workaround nonetheless.

**Reply by imbwaldo** [reply]
Oh, a completely different approach—a different set of conditions. No longer need to test for an image.
